### PR TITLE
Private Links | Endpoints Aura Enterprise Docs | Network Access rewording

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

File: modules/ROOT/pages/platform/security.adoc [hunk 1]
```diff
@@ -12,13 +12,27 @@ By default, each cloud provider encrypts all backup buckets (including the objec
...
 label:AuraDS-Enterprise[]
 
-AuraDB Enterprise and AuraDS Enterprise run within a Virtual Private Cloud (VPC) isolation for your deployment.
+AuraDB Enterprise and AuraDS Enterprise run in a dedicated cloud Account (AWS), Subscription (Azure) or Project (GCP) to achieve complete isolation for your deployment.
 
-The VPC enables you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
+Additional VPC boundaries enable you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
 
-Please note that the Aura Console runs in a separate VPC.
+Please note that the Aura Console runs in a separate VPC, separate from the rest of Aura.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:19]:
```suggestion
Note that the Aura Console runs in a separate VPC, separate from the rest of Aura.
```

File: modules/ROOT/pages/platform/security.adoc [hunk 2]
```diff
@@ -12,13 +12,27 @@ By default, each cloud provider encrypts all backup buckets (including the objec
...
 
-=== AWS Private endpoints
+== Network Access
+
+Neo4j provides authorized users with the ability to configure certain aspects of the infrastructure that is used to run and manage Aura instances, and the networking the platform leverages to establish secure connections between the database (Aura) and application VPCs.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+You can also choose whether traffic to your Aura instance is allowed to traverse the public internet, or if routes should be restricted to the private link by disabling public traffic. 
+To enable this, customers need to create a private endpoint in their own VPC.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:27]: I'm not sure about the Aura docs, but we tend not to refer to our users as _customers_. It sounds like marketingese.

**Review by AlexicaWright — COMMENTED** [review]
Just a small thought.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

File: modules/ROOT/pages/platform/security.adoc [hunk 3]
```diff
@@ -6,13 +6,27 @@
...
-The VPC enables you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
+Additional VPC boundaries enable you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
 
-Please note that the Aura Console runs in a separate VPC.
+Note that the Aura Console runs in a separate VPC, separate from the rest of Aura.
 
-=== AWS Private endpoints
+== Network Access
+
+Neo4j provides authorized users with the ability to configure certain aspects of the infrastructure that is used to run and manage Aura instances, and the networking the platform leverages to establish secure connections between the database (Aura) and application VPCs.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:17]:
```suggestion
Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
```

To sound a little less marketingy and also way less complicated.

File: modules/ROOT/pages/platform/security.adoc [hunk 4]
```diff
@@ -6,13 +6,27 @@
...
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+You can also choose whether traffic to your Aura instance is allowed to traverse the public internet, or if routes should be restricted to the private link by disabling public traffic. 
+To enable this, create a private endpoint in your own VPC.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:21]:
```suggestion
You can also choose whether traffic to your Aura instance is allowed to traverse the public internet, or if routes should be restricted to the private link by disabling public traffic. 
To enable this, create a private endpoint in your own VPC. 
```
To enable what? Traverse the public internet or restrict traffic?

**Review by AlexicaWright — COMMENTED** [review]
Some comments.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

File: modules/ROOT/pages/platform/security.adoc (hunk 4 above)
Comment (fiquick) [reply modules/ROOT/pages/platform/security.adoc:21]: In order to control traffic and restrictions, customers will need to create a private endpoint in their own VPC.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

File: modules/ROOT/pages/platform/security.adoc [hunk 5]
```diff
@@ -6,13 +6,29 @@
...
-=== AWS Private endpoints
+== Network Access
+
+Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+You can also choose whether traffic to your Aura instance is allowed to traverse the public internet or instead uses the cloud provider's internal network. 
+To use the cloud provider's network, you can create a private endpoint inside your VPC. 
+This gives you a totally private connection to Aura.
+It's your choice if you want to prevent your Aura instances from being publicly accessible or if you want both public and private access.
```
Comment (fiquick) [inline modules/ROOT/pages/platform/security.adoc:23]: `` updated content here

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

File: modules/ROOT/pages/platform/security.adoc [hunk 6]
```diff
@@ -6,13 +6,29 @@
...
-Please note that the Aura Console runs in a separate VPC.
+Note that the Aura Console runs in a separate VPC, separate from the rest of Aura.
 
-=== AWS Private endpoints
+== Network Access
+
+Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+You can also choose whether traffic to your Aura instance is allowed to traverse the public internet or instead uses the cloud provider's internal network.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:20]:
```suggestion
You can also choose whether traffic to your Aura instance is allowed to traverse the public internet or if it should use the cloud provider's internal network. 
```

File: modules/ROOT/pages/platform/security.adoc [hunk 7]
```diff
@@ -6,13 +6,29 @@
...
 
-=== AWS Private endpoints
+== Network Access
+
+Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+You can also choose whether traffic to your Aura instance is allowed to traverse the public internet or instead uses the cloud provider's internal network. 
+To use the cloud provider's network, create a private endpoint inside your VPC. 
+This gives you a totally private connection to Aura.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:22]:
```suggestion
This gives you a private connection to Aura.
```

File: modules/ROOT/pages/platform/security.adoc [hunk 8]
```diff
@@ -6,13 +6,29 @@
...
+It's your choice if you want to prevent your Aura instances from being publicly accessible or if you want both public and private access.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:23]: I suggest to rewrite this section to highlight the following:
* If you want your Aura instances to be private, do this.
* If you want your Aura instances to be both private and public, do this.
* If you want your Aura instances to be public, do this.

If the traffic is allowed to traverse the public internet, does that mean the instance is publicly accessible?
And if the traffic is routed through the cloud provider's network instead, is it then completely private? What do I need to do to allow both public and private access?

File: modules/ROOT/pages/platform/security.adoc [hunk 9]
```diff
@@ -6,13 +6,29 @@
...
+
+== Private endpoints
+
+Private endpoints are network interfaces inside your own VPC, which can only accessed within your private network.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:27]:
```suggestion
Private endpoints are network interfaces inside your own VPC, which can only be accessed within your private network. 
```

**Review by AlexicaWright — COMMENTED** [review]
Also, on line 50, remove the word _we_. Like this maybe:
_In AWS region us-east-1, the Availability Zone with  use1-az3 is not supported for private endpoints._

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

File: modules/ROOT/pages/platform/security.adoc (hunk 8 above)
Comment (fiquick) [reply modules/ROOT/pages/platform/security.adoc:23]: @afharvey Hi Alex, please could I bring you in on this conversation to answer Jessica's questions?

File: modules/ROOT/pages/platform/security.adoc [hunk 10]
```diff
@@ -6,13 +6,29 @@
...
-=== AWS Private endpoints
+== Network Access
+
+Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+You can also choose whether traffic to your Aura instance is allowed to traverse the public internet or if it should use the cloud provider's internal network. 
+To use the cloud provider's network, create a private endpoint inside your VPC. 
+This gives you a private connection to Aura.
+It's your choice if you want to prevent your Aura instances from being publicly accessible or if you want both public and private access.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:23]: Can't push my suggestion so here it is:
```suggestion
An Aura instance can be publicly available, completely private, or both.

To configure this, you need to be authorized to access to the part of the infrastructure that runs and handles these as well as the networking used to establish secure connections between the database and the application's VPC.
This includes the ability to connect over the cloud provider's private link and private endpoint.

You can choose whether traffic to your Aura instance is allowed to traverse the public internet or if it should use the cloud provider's internal network.
If you enable public traffic, the instance is publicly accessible by anyone with the correct username and password.

For your instance to instead be completely private, use the cloud provider's network and create a private endpoint inside your VPC, which gives you a private connection to Aura.

If you enable *both* private and public traffic, your instance is accessible both privately and publicly.
```

**Review by AlexicaWright — COMMENTED** [review]
@afharvey could you verify that my suggestion isn't telling lies?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

File: modules/ROOT/pages/platform/security.adoc (hunk 10 above)
Comment (fiquick) [reply modules/ROOT/pages/platform/security.adoc]: Thanks Jessica - I also had a go at re-writing it, wdyt?

```
== Network Access

Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
This includes the ability to connect over the cloud provider's private link and private endpoint.

* If you want your Aura instances to be both private and public, go to the *Aura Console* > *Security* > *Network Access*, then in the network configuration set public and private access.
* If you want your Aura instances to be public, traffic to your Aura instance is allowed to traverse the public internet and the instance is accessible with the username and password. 
Anyone in the world can try to log in if they know the public address.
* If you want your Aura instances to be private, use your cloud provider's internal network. 
To use the cloud provider's network, create a private endpoint inside your VPC. 
This gives you a private connection to Aura. 
If public traffic is turned off then access is completely private and the only way to connect to your database is from inside your network (your VPC in your AWS/Azure/GCP account) using an internal IP address you choose and DNS records you create. 

== Private endpoints

Private endpoints are network interfaces inside your own VPC, which can only be accessed within your private network. 
The cloud provider connects them over their network to Neo4j Aura. 
By design they are not exposed to the public internet, ensuring that critical services are accessible only through private, secure networks.
```

Comment (fiquick) [reply modules/ROOT/pages/platform/security.adoc]: On second thoughts, I think a mix of both edits could be nice

File: modules/ROOT/pages/platform/security.adoc [hunk 11]
```diff
@@ -6,13 +6,32 @@
...
-Please note that the Aura Console runs in a separate VPC.
+Note that the Aura Console runs in a separate VPC, separate from the rest of Aura.
 
-=== AWS Private endpoints
+== Network access
+
+Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+An Aura instance can be publicly available, completely private, or both. To configure this go to the *Aura Console* > *Security* > *Network Access*.
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply modules/ROOT/pages/platform/security.adoc:20]: Line 17 and line 21 essentially say the same thing. Also, I would recommend avoiding talking about _Authorized users_ and then _you_ a few lines below. So I stand by my original suggestion for the first section:
_An Aura instance can be publicly available, completely private, or both.
To configure this, you need to be authorized to access the part of the infrastructure that runs and handles these instances as well as the networking used to establish secure connections between the database and the application's VPC.
This includes the ability to connect over the cloud provider's private link and private endpoint._

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

File: modules/ROOT/pages/platform/security.adoc (hunk 11 above)
[the remark this answers is not in the document]
Comment (fiquick) [reply modules/ROOT/pages/platform/security.adoc]: Your suggestion does sounds good - I've edited the first section to include it 👍

File: modules/ROOT/pages/platform/security.adoc [hunk 12]
```diff
@@ -6,13 +6,32 @@
...
+An Aura instance can be publicly available, completely private, or both. 
+To configure this, you need to be authorized to access the part of the infrastructure that runs and handles these instances as well as the networking used to establish secure connections between the database and the application's VPC.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+* If your Aura instances are public, traffic to them is allowed to traverse the public internet and they are accessible with the correct username and password. 
+
+* For your instance to be completely private, turn public traffic off, use the cloud provider's network, and create a private endpoint inside your VPC, which gives you a private connection to Aura.
+The only way to connect to your database is from inside your network (your VPC in your AWS/Azure/GCP account) using an internal IP address you choose and DNS records you create. 
+
+To choose network access settings go to *Aura Console* > *Security* > *Network Access*.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:26]:
```suggestion
To select network access settings go to *Aura Console* > *Security* > *Network Access*.
```

File: modules/ROOT/pages/platform/security.adoc [hunk 13]
```diff
@@ -6,13 +6,32 @@
...
+== Network access
+
+An Aura instance can be publicly available, completely private, or both. 
+To configure this, you need to be authorized to access the part of the infrastructure that runs and handles these instances as well as the networking used to establish secure connections between the database and the application's VPC.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+* If your Aura instances are public, traffic to them is allowed to traverse the public internet and they are accessible with the correct username and password. 
+
+* For your instance to be completely private, turn public traffic off, use the cloud provider's network, and create a private endpoint inside your VPC, which gives you a private connection to Aura.
+The only way to connect to your database is from inside your network (your VPC in your AWS/Azure/GCP account) using an internal IP address you choose and DNS records you create.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security.adoc:24]: For a bullet list to make sense here, both points need to follow the same structure. Right now, the first point informs you while the second one instructs. Maybe you could remove the bullets and keep the wording?
You could also rewrite to make them both instructions _or_ information, but the wording is good now so I'd say just remove the bullets.

**Review by AlexicaWright — COMMENTED** [review]
Almost there! I like this!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-331.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-331.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Security](https://neo4j-docs-aura-331.surge.sh/aura/platform/security)

**Review by AlexicaWright — APPROVED** [review]
👍 LGTM! Well done and thank you Fi!

**Comment by fiquick** [comment]
[Change Docs from VPC Isolation to Network Access](https://trello.com/c/EQOx4AKY/277-change-docs-from-vpc-isolation-to-network-access)

**Pull request merged**